### ISSUE TITLE
is redcon (go) actually more performant than redcon.c?

### ISSUE THREAD

**Issue by a12one** [tidwall/redcon.c#1]
if I read the benchmark correctly? this is interesting result.

**Comment by tidwall** [comment]
I don't think so. The C version should be faster, but there would need to be a benchmark comparing the two to be certain.
